index.html

```diff
@@ -2428,58 +2428,71 @@
         </ol>
       </section>
     </section>
-    <section class='informative'>
-      <h2>
-        Security Considerations
-      </h2>
-      <p class="ednote">
-        This section is a placeholder to record security considerations as we
-        gather them through working group discussion.
-      </p>
-      <section>
+    <section class="informative">
```

Comment:
Each item was so small as to beg the question why would a top level section be in place for them.  Moreover, as they are all related to W3C horizontal review requirements, it felt sensible to gather them together to me.